test/e2e/storage/subpath.go

```diff
@@ -160,15 +160,27 @@ var _ = utils.SIGDescribe("Subpath", func() {
 				vol.cleanupVolume(f)
 			})
 
-			It("should support non-existent path", func() {
+			// Containerized kubelet does not support creation of
+			// directories/files outside of /var/lib/kubelet, some tests must
```

Comment:
Can you modify the hostpath test to use a path inside /var/lib/kubelet?

Reply:
Actually wait, even in containerized kubelet, hostpath should still work outside of /var/lib/kubelet.

Reply:
It works outside of /var/lib/kubelet, but it can't create files/directories there. /rootfs is read only and there is no safe mkdir via nsenter. Do I miss anything?

Reply:
Didn't you make a change to make /rootfs rw?

Reply:
No, it's read-only in hack/local-up-cluster.sh (and in OpenShift)